### PR TITLE
Mobile layout cleaned up.

### PR DESCRIPTION
**Pull request by gabriel3000**

### CODE REVIEW

**Comment by gabriel3000** [comment]
I originally submitted this pull request to the master branch. Please disregard that pull request and consider this one instead. Thanks!

File: css/generated/style.css [hunk 1]
```diff
@@ -149,6 +149,13 @@ body {
   color: #444444;
   border-top: 5px solid #f23109; }
 
+body.tablet {
+  min-width: 320px;
+  border-top: 90px solid #f5f5f5; }
+
+body.mobile.tablet {
```
Comment (quoo) [inline css/generated/style.css]: Couldn't this just be body.mobile?

File: scss/global/_base.scss [hunk 2]
```diff
@@ -123,6 +123,13 @@ body {
 	color: $default-2-text;
 	border-top: 5px solid $isobar-orange;
 }
+body.tablet{
+	min-width:320px;
+	border-top: 90px solid #f5f5f5;
+}
+body.mobile.tablet{
```
Comment (quoo) [inline scss/global/_base.scss]: Couldn't this just be body.mobile?

File: scss/global/_base.scss [hunk 3]
```diff
@@ -123,6 +123,13 @@ body {
...
+	border-top: 60px solid #f5f5f5;
```
Comment (quoo) [inline scss/global/_base.scss]: Please use the color variables.

**Pull request merged**